ISISNeutronMuon/MDANSE
# Add a backup solution if unit cell is not defined/usable

**Description of work**
In case no unit cell was defined for a system (or the cell is set to 0), calculates the maximum r in DistHistCutoffConfgurator from the span of atom coordinates.

This does **not** mean that the analysis results will be correct. For now, this PR will stop the GUI from crashing.

closes #516

**Fixes**
1. Added a backup calculation of maximum r for DistHistCutoffConfigurator.
2. Added a preview of the first unit cell in trajectory view.

**To test**
Load a trajectory without a unit cell (Gaussian, xtb, Dmol3, ...?). Check that the GUI does not crash when Pair Distribution Function or Static Structure Factor are selected.
